CSLabs.Api/Models/Enums/EBannerTypes.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace CSLabs.Api.Models.Enums
+{
+    public enum EBannerTypes
```

Comment:
rename to `EBannerType`.  As a general rule of thumb, classes and enums should be singular (not plural).